src/platform/windows/installing.md

```diff
@@ -24,11 +24,11 @@ desktop to start Anki.
 
 ## Upgrading
 
-If upgrading from Anki 2.1.6+, there is no need to uninstall the previous
+If upgrading from Anki 2.1.06+, there is no need to uninstall the previous
```

Comment:
Thanks for the info, for those of us not familiarized with these aspects, it can be confusing (I was searching 20 minutes before resolving the mistery), but it's just a minor detail for 99% od the users, I guess

Reply:
When it was released, it was called "2.1.6", not "2.1.06", so I suggest we stick with the first one :-)

Reply:
Yes, sorry, I forgot to change it back, haha. Done!